### ISSUE TITLE
new Audio() - taking over the iPhone, muting everything

### ISSUE THREAD

**Issue by Andarist** [goldfire/howler.js#608]
Do you know if there is any workaround for this issue?

For example while having Spotify in the background just creating `new Audio()` is taking over completely the sound on the device and muting the background music.

And there is made a test during Howler initialization using this constructor - https://github.com/goldfire/howler.js/blob/master/dist/howler.js#L203

I have no idea what's going on, debugging this for over a day. Example site which suffers from it - www.williammalone.com/posts/ios-9-web-audio-api-safari-mobile-issue/

**Comment by goldfire** [comment]
As far as I am aware, there can only be one source of audio (app/site) active at any time on iPhone, and this isn't something that can be controlled. Can anyone else confirm this?

**Comment by Andarist** [comment]
From what I got told by my colleague in native apps you can define if you want to just play a sound or if you want to 'take over'. But in this case I'm not even playing sound... yet. Just testing codecs. And also hijacking like this

```
window.Audio = function(){};
window.Audio.prototype.canPlayType = function(){return 'probably'};
```

and later restoring after the test is done seems to be 'fixing' the issue. Sound in the background is not muted and sound from the website can be played at the same time.

**Comment by Andarist** [comment]
Ok, I *think* now that I got to the bottom of this.

Thats the weirdest thing I've encountered for a long time, tested it now on several sites. Just entering them with my iPhone plugged in so I can hook to them from DevTools in Safari on my Mac.

So a combo of:
```
(new webkitAudioContext()).createGain()
new Audio()
```

seems to be the culprit and the order of those operation may be the opposite it can be:
```
new Audio()
(new webkitAudioContext()).createGain()
```

And after the second one background music will just fade out and mute itself.
